cbmc/litani.nix

```diff
@@ -4,7 +4,7 @@
 , fetchFromGitHub
 }:
 
-stdenvNoCC.mkDerivation rec {
+stdenvNoCC.mkDerivation {
```

Comment:
What's this about?

Reply:
`rec` stands for recursive attribute set, which allows the attributes in the attribute set to refer to each other. 
Since in the attribute set, we are not self referencing, it's actually redundant, and my linter was complaining, so I removed it by the way.